apps/dg/components/case_card/case_card.js

```diff
@@ -308,16 +309,65 @@ DG.React.ready(function () {
                 tHasFormula = iAttr.get('hasFormula'),
                 tFormula = iAttr.get('formula'),
                 tCase = iShouldSummarize ? null : (iChildmostSelected && iChildmostSelected[0]) || iCases[0],
-                tValue = iShouldSummarize ? '' : tCase && tCase.getValue(tAttrID);
+                tValue = iShouldSummarize ? '' : tCase && tCase.getValue(tAttrID),
+                tType = iAttr.get('type');
             this.state.attrIndex++;
```

Comment:
You also need:
          if( cellValue && cellValue.jsonBoundaryObject)
            type = 'boundary';

Reply:
Right.  This seems to fix it (tweaked for local vars):
if( tValue && tValue.jsonBoundaryObject)
  tType = 'boundary';		

Changes have been added.